tests/system/action/user/test_email_client_server.py

```diff
@@ -0,0 +1,510 @@
+import os
```

Comment:
The location of this file does not fit; it is something more general since it only tests the email mixin.

Reply:
moved to tests/system/action and renamed to test_send_email_mixin.py